Header header

uint32 lights_high
uint32 lights_medium
uint32 lights_left
uint32 lights_right
uint32 lights_danger

uint32 ignition

uint32 horn

uint32 throttle_pressure
uint32 brake_pressure
uint32 clutch_pressure
